clientd3d/game.c

```diff
@@ -230,7 +230,7 @@ void SetPlayerRemoteView(ID objID, int flags, int height, BYTE light)
       if (flags & REMOTE_VIEW_VALID_HEIGHT)
 	 player.viewHeight = floor + height;
       else if (flags & REMOTE_VIEW_TOP)
-	 player.viewHeight = viewObject->motion.z + objHeight;
+	 player.viewHeight = viewObject->motion.z - 10;
```

Comment:
This can't be the right fix, as it puts REMOTE_VIEW_TOP below both REMOTE_VIEW_BOTTOM and REMOTE_VIEW_MID.  One alternative would be to go to the Blakod for the globe and have it send REMOTE_VIEW_BOTTOM or MID instead.  But in any case you'll need to check all the other globes in the game to make sure this doesn't break them.

Reply:
Or, maybe move the globes down a little bit.

Reply:
My mistake. I think my brain was mush in this moment. My new approach is to use `REMOTE_VIEW_BOTTOM` rather than `REMOTE_VIEW_TOP`, setting the players viewpoint to be the bottom of the globe rather than the very top.